db/migrate/20170511120047_remove_container_group_failed_sync_event.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveContainerGroupFailedSyncEvent < ActiveRecord::Migration[5.0]
+  class Relationship < ActiveRecord::Base; end
+
+  class MiqEventDefinition < ActiveRecord::Base; end
```

Comment:
@zakiva You are correct but as I recall the idea was to future-proof the migrations in case the model may introduce sub-classing through the `type` column.